src/libs/functions/openApp.js

```diff
@@ -41,6 +41,35 @@ const slugFallbacks = {
   }
 }
 
+/**
+ * Manage custom behavior for mobile app
+ * Open the native mobile app if no custom behavior
+ * @param {import("cozy-client/dist/index").CozyClient} client - CozyClient instance
+ * @param {any} navigation - The React navigation context
+ * @param {string} href - The app web url
+ * @param {AppManifest} app - The app information
+ * @returns {Promise}
+ */
+const mobileDispatcher = async (client, navigation, href, app, iconParams) => {
+  if (app.slug === 'passwords') {
+    try {
+      const { extension_installed } = await client
+        .getStackClient()
+        .fetchJSON('GET', '/data/io.cozy.settings/io.cozy.settings.bitwarden')
```

Comment:
On cozy-pass-web, we use fetchJSON directly like this to fetch this value and cache is not wanted.

Reply:
Yeah, I don't think we should by pass cozy-client redux store. In fact, I'm pretty sure we already do this kind of request during the login phase, and that we'll need this information later when we'll deal with the vault. 

So we should use the "cache" of cozy-client here. 

And since you are in a non react world, I would suggest to not use client.query() with a fetchPolicy (see https://github.com/cozy/cozy-client/issues/931) but instead `fetchQueryAndGetFromState` (see for exemple https://github.com/cozy/cozy-settings/commit/24be85a43c185308075978ce38b81862f2b518f5)

Reply:
Will need https://github.com/cozy/cozy-client/pull/1339